docs/pages/router/basics/real-life-nav.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Navigation Patterns
+description: Apply Expo Router basics to real-life navigation patterns used in your app.
+sidebar_title: Common Patterns
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+Now that we know the basics of how files and folders are named and arranged in Expo Router, let's apply that knowledge, looking at some real-life navigation patterns you might use in your app.
+
+## Stacks inside tabs: nested navigators
+
+If the typical starting point for your app is a set of tabs, but one or more tabs may have more than one screen associated with it, nesting a stack navigator inside of a tab is often the way to go. This pattern often results in intuitive URL's and scales well to desktop web apps, where the primary tabs are often always visible.
+
+Consider this navigation tree:
+
+<FileTree
+  files={[
+    ['app/(tabs)/\_layout.tsx'],
+    ['app/(tabs)/index.tsx', 'single page tab'],
+    ['app/(tabs)/feed/_layout.tsx', 'tab with a stack inside'],
+    ['app/(tabs)/feed/index.tsx'],
+    ['app/(tabs)/feed/[postId].tsx'],
+    ['app/(tabs)/settings.tsx', 'single page tab'],
+  ]}
+/>
+
+In the **app/(tabs)\_layout.tsx** file, return a `Tabs` component:
+
+```tsx _layout.tsx
+import { Tabs } from 'expo-router';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tab.Screen name="index" options={{ title: 'Home' }} />
+      <Tab.Screen name="feed" options={{ title: 'Feed' }} />
+      <Tab.Screen name="settings" options={{ title: 'Settings' }} />
+    </Tabs>
+  );
+}
+```
+
+In the **app/(tabs)/feed/\_layout.tsx** file, return a `Stack` component:
+
+```tsx feed/_layout.tsx
+import { Stack } from 'expo-router';
+
+/* @info Setting initialRouteName ensures that direct links deep into the stack still push the index route onto the stack first. */
+export const unstable_settings = {
+  initialRouteName: 'index',
+};
+/* @end */
+
+export default function FeedLayout() {
+  return <Stack />;
+}
+```
+
+Now, within the **app/(tabs)/feed** folder, you can have `Link` components that point to different posts (e.g., `/feed/123`). Those links will push the `feed/[postId]` route onto the stack, leaving the tab navigator visible.
+
+You can also navigate from any other tab to a post in the feed tab with the same URL. Use `withAnchor` in conjunction with `initialRouteName` to ensure that the `feed/index` route is always the first screen in the stack:
+
+```tsx app/(tabs)/feed/index.tsx
+<Link href="/feed/123" withAnchor>
+  Go to post
+</Link>
+```
+
+You can also nest tabs inside of an outer stack navigator. That is often more useful for displaying modals over the tabs.
```

Comment:
This would be a good place to add a note that every navigator adds a header so you'd usually want to hide the outer ones.

Reply:
Tried a tooltip for this to start, so I could put it in the example tab layout, but could spell it out elsewhere